lib/internal/repl/experimental/index.js

```diff
@@ -0,0 +1,635 @@
+'use strict';
+
+const {
+  ArrayFrom,
+  ArrayPrototypeFilter,
+  ArrayPrototypeFind,
+  ArrayPrototypeForEach,
+  ArrayPrototypeIncludes,
+  ArrayPrototypeMap,
+  FunctionPrototypeBind,
+  MathFloor,
+  ObjectDefineProperty,
+  ObjectGetOwnPropertyDescriptor,
+  ObjectGetOwnPropertyNames,
+  ReflectApply,
+  RegExpPrototypeExec,
+  RegExpPrototypeSymbolSplit,
+  SafeMap,
+  StringPrototypeIndexOf,
+  StringPrototypeRepeat,
+  StringPrototypeSlice,
+  StringPrototypeStartsWith,
+  StringPrototypeSubstring,
+  Symbol,
+} = primordials;
+
+const { createInterface, clearScreenDown } = require('readline');
+const { parse: acornParse } = require('internal/deps/acorn/acorn/dist/acorn');
+const walk = require('internal/deps/acorn/acorn-walk/dist/walk');
+const {
+  isIdentifier,
+  strEscape,
+  underlineIgnoreANSI,
+  isStaticIdentifier,
+  isScopeOpen,
+  stripVTControlCharacters: stripAnsi,
+} = require('internal/repl/experimental/util');
+const highlight = require('internal/repl/experimental/highlight');
+const getHistory = require('internal/repl/experimental/history');
+const util = require('util');
```

Comment:
`require('util')` can be modified at runtime by users.
```suggestion
const { styleText } = require('util');
```

Reply:
(Resolved locally)